UmbracoV9/UmbracoV9.csproj

```diff
@@ -46,5 +48,8 @@
         <RazorCompileOnBuild>false</RazorCompileOnBuild>
         <RazorCompileOnPublish>false</RazorCompileOnPublish>
     </PropertyGroup>
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+		<Exec Command="xcopy $(SolutionDir)Iconic\App_Plugins\Iconic\*.* $(ProjectDir)App_Plugins\Iconic\ /S /E /Y" />
+	</Target>
```

Comment:
The targets file that you removed a few lines above copies the package's files into the V9 project for testing already.

Reply:
true - both changes can be reverted...